pyintegration/deephaven/csv.py

```diff
@@ -15,12 +15,17 @@
 _JCsvTools = jpy.get_type("io.deephaven.csv.CsvTools")
 _JParsers = jpy.get_type("io.deephaven.csv.parsers.Parsers")
 _JArrays = jpy.get_type("java.util.Arrays")
-
+_JQueryConstants = jpy.get_type("io.deephaven.util.QueryConstants")
 
 def read(
     path: str,
     header: Dict[str, dht.DType] = None,
     headless: bool = False,
+    skip_rows: int = 0,
+    num_rows: int = _JQueryConstants.MAX_LONG,
```

Comment:
should this be max_rows?

Reply:
I think max_long so it is consistent with the groovy default.

Reply:
i meant 'num_rows'